packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -107,6 +107,15 @@ class _PreferredAppBarSize extends Size {
 /// ** See code in examples/api/lib/material/app_bar/app_bar.0.dart **
 /// {@end-tool}
 ///
+/// Material Design 3 introduced new types of app bar.
+/// {@tool dartpad}
+/// This sample shows the creation of [AppBar] widget with [shadowColor] and
+/// [scrolledUnderElevation] properties, as described in:
+/// https://m3.material.io/components/top-app-bar/overview
+///
+/// ** See code in examples/api/lib/material/floating_action_button/floating_action_button.2.dart **
```

Comment:
Uhm, I think this is the wrong link, although it does create an `AppBar`. It would seem weird to me to have it say "FloatingActionButton Sample" in the sample for the `AppBar`.